pom.xml

```diff
@@ -222,7 +220,7 @@
         <dependency>
             <groupId>io.jenkins.tools.bom</groupId>
             <artifactId>bom-${jenkins.baseline}.x</artifactId>
-            <version>2961.v1f472390972e</version>
+            <version>3893.v213a_42768d35</version>
```

Comment:
Not the very latest BOM release, but one that contains several recent EE 9 updates to plugins.